openfga/v1/openfga_service.proto

```diff
@@ -553,6 +553,29 @@ service OpenFGAService {
     };
   }
 
+  rpc RenameStore(RenameStoreRequest) returns (RenameStoreResponse) {
```

Comment:
Instead of a `RenameStore` being a `POST` on `/stores/{store_id}/rename`

Maybe we can call it `UpdateStore` and make it a `PATCH` on `/stores/{store_id}`?

Reply:
Yeah, of course!

Reply:
Sorry, I just realized I missed this part in the first comment by @rhamzeh. I renamed the request to `UpdateStore` (also the `Request` and `Response` message).